fendermint/app/settings/src/lib.rs

```diff
@@ -93,6 +98,54 @@ pub struct BroadcastSettings {
     pub max_retries: u8,
 }
 
+#[serde_as]
+#[derive(Debug, Deserialize, Clone)]
+pub struct TopDownConfig {
+    /// The number of blocks to delay before reporting a height as final on the parent chain.
+    /// To propose a certain number of epochs delayed from the latest height, we see to be
+    /// conservative and avoid other from rejecting the proposal because they don't see the
+    /// height as final yet.
+    pub chain_head_delay: BlockHeight,
+    /// Parent syncing cron period, in seconds
+    #[serde_as(as = "DurationSeconds<u64>")]
+    pub polling_interval_secs: Duration,
```

Comment:
```suggestion
    pub polling_interval: Duration,
```
No need for saying anything about units if it's `Duration`.

Reply:
fixed.